.circleci/config.yml

```diff
@@ -65,39 +62,11 @@ jobs:
             elif [[ "$branch" != "trunk" && "$branch" != "develop" ]]; then
               # This happens on the first push of a new branch, when there
               # isn't a PR open for it yet.
-              echo "Running on a feature branch with no open PR: skipping Bintray upload"
+              echo "Running on a feature branch with no open PR: skipping S3 upload"
               exit 0
             fi
 
-            # Locally, calling `./gradlew bintrayUpload` is enough to generate
-            # all the required artifacts. On CI, unless run each task
-            # individually, the Bintray task will fail to find the AAR and
-            # POM artifacts in the build/ folder.
-            #
-            # This builds the AAR.
-            ./gradlew --stacktrace assembleRelease
-
-            # This builds the POM.
-            #
-            # On local, the POM is generated automatically as part of
-            # assembleRelease, but that doesn't seem to be the case on CI. Here
-            # we need to explicitly generated it, otherwise bintrayUpload won't
-            # find it and won't upload it.
-            #
-            # Worth noting that the Bintray plugin mentions the necessity for
-            # some workarounds to generate the POM under certain circumstances,
-            # although none of them apply to our current usage. See:
-            # https://github.com/bintray/gradle-bintray-plugin/tree/67718c3a65b64dbfe7534a82a178da2c57153a5d#maven-publications
-            #
-            # See also the description and comments in this PR:
-            # https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/39
-            #
-            # Notice the `-PbintrayVersion` parameter to make sure the POM has
-            # the same version as the one bintrayUpload will use.
-            ./gradlew --stacktrace generatePomFileForUtilsPublicationPublication -PbintrayVersion=$PREFIX
-
-            # Finally, this uploads both AAR and POM to Bintray.
-            ./gradlew --stacktrace bintrayUpload -PbintrayVersion=$PREFIX
+            ./gradlew --stacktrace publish -PpublishVersion=$PREFIX
```

Comment:
I wish the task posted a confirmation message, like "Successfully uploaded to `URL` as `$PREFIX`. I would simplify finding out what version string to use in the consumer.

![image](https://user-images.githubusercontent.com/1218433/112774654-0ba03c00-9086-11eb-9f0e-fea540df4444.png)

Reply:
Agree with this – adding `echo $PREFIX` would even be enough?

Reply:
Thank you both for the review! Good idea to add a message for this 👍 

I am looking at a few other things that might impact this PR, so just wanted to give you a heads up that it might take me a little bit more to come back to it.

Reply:
This has been addressed in https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/67.